### PR TITLE
Add Lit 2 release livestream post

### PR DESCRIPTION
**Pull request by arthurevans**

### CODE REVIEW

**Comment by arthurevans** [comment]
Not sure why the release blog post shows up here as well, even though it's already merged...

**Comment by aomarks** [comment]
> Not sure why the release blog post shows up here as well, even though it's already merged...

Maybe need to pull your main branch?

**Review by aomarks — APPROVED** [review]

File: packages/lit-dev-content/site/blog/2021-09-22-lit-2-release-livestream.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+tags: blogPosts
+layout: blog-post.html
+title: "Watch the Lit 2 release livestream"
+summary: "What's new in Lit 2, a big-picture view of Lit usage at Google, and a community panel discussion."
+date: 2021-09-22
+---
+
+<div class="preamble">
+
+If weren't able to join us for the Lit 2 the release livestream, you can watch the recording on YouTube.
```
Comment (AndrewJakubowicz) [inline packages/lit-dev-content/site/blog/2021-09-22-lit-2-release-livestream.md:11]: Nit: Add `you`: "If **you** weren't"

Remove `the` from "Lit 2 ~~the~~ release"

**Review by AndrewJakubowicz — APPROVED** [review]
Two small typo nits (see comment).

LGTM!

File: packages/lit-dev-content/site/blog/2021-09-22-lit-2-release-livestream.md (hunk 1 above)
Comment (arthurevans) [reply packages/lit-dev-content/site/blog/2021-09-22-lit-2-release-livestream.md]: Fixed, thanks!

**Pull request merged**